service/src/main/resources/application.yml

```diff
@@ -30,6 +30,9 @@ gpc-consumer:
     structuredPath: ${GPC_CONSUMER_GPC_STRUCTURED_PATH:/*/STU3/1/gpconnect/fhir/Patient/$gpc.getstructuredrecord}
     findPatientPath: ${GPC_CONSUMER_GPC_GET_PATIENT_PATH:/*/STU3/1/gpconnect/documents/Patient}
     documentPath: ${GPC_CONSUMER_GPC_GET_DOCUMENT_PATH:/*/STU3/1/gpconnect/documents/Binary/**}
+    structuredFhirBasePathRegex: (.*?)\/STU3/1/gpconnect/(.*?)\/
+    sspDomain: ${GPCC_SSP_FQDN:proxy.opentest.hscic.gov.uk}
+    sspEnabled: ${GPC_TLS_ENABLED:false}
```

Comment:
These variables will needed added to the readme

Reply:
GPC_TLS_ENABLED removed, GPCC_SSP_FQDN renamed to GPC_CONSUMER_SSP_FQDN and is in readme